eng/pipelines/common/variables.yml

```diff
@@ -20,26 +20,26 @@ variables:
 - name: isWasmOnlyBuild
   value: ${{ in(variables['Build.DefinitionName'], 'runtime-wasm', 'runtime-wasm-libtests', 'runtime-wasm-non-libtests') }}
 - name: isiOSLikeOnlyBuild
-  value: ${{ in(variables['Build.DefinitionName'], 'runtime-ios', 'runtime-tvos', 'runtime-ioslike') }}
+  value: ${{ in(variables['Build.DefinitionName'], 'runtime-ioslike') }}
 - name: isiOSLikeSimulatorOnlyBuild
-  value: ${{ in(variables['Build.DefinitionName'], 'runtime-iossimulator', 'runtime-tvossimulator', 'runtime-ioslikesimulator') }}
+  value: ${{ in(variables['Build.DefinitionName'], 'runtime-ioslikesimulator') }}
 - name: isAndroidOnlyBuild
   value: ${{ in(variables['Build.DefinitionName'], 'runtime-android') }}
 - name: isAndroidEmulatorOnlyBuild
   value: ${{ in(variables['Build.DefinitionName'], 'runtime-androidemulator') }}
 - name: isMacCatalystOnlyBuild
-  value: ${{ in(variables['Build.DefinitionName'], 'runtime-catalyst', 'runtime-maccatalyst') }}
+  value: ${{ in(variables['Build.DefinitionName'], 'runtime-maccatalyst') }}
 - name: isLinuxBionicOnlyBuild
-  value: ${{ in(variables['Build.DefinitionName'], 'runtime-linuxbionic', 'runtime-bionic') }}
+  value: ${{ in(variables['Build.DefinitionName'], 'runtime-linuxbionic') }}
 - name: isRunSmokeTestsOnly
-  value: ${{ notin(variables['Build.DefinitionName'], 'runtime-wasm', 'runtime-wasm-libtests', 'runtime-wasm-non-libtests', 'runtime-ios', 'runtime-tvos', 'runtime-ioslike', 'runtime-iossimulator', 'runtime-tvossimulator', 'runtime-ioslikesimulator', 'runtime-android', 'runtime-androidemulator', 'runtime-catalyst', 'runtime-maccatalyst', 'runtime-linuxbionic', 'runtime-bionic') }}
+  value: ${{ notin(variables['Build.DefinitionName'], 'runtime-extra-platforms', 'runtime-wasm', 'runtime-wasm-libtests', 'runtime-wasm-non-libtests', 'runtime-ioslike', 'runtime-ioslikesimulator', 'runtime-android', 'runtime-androidemulator', 'runtime-maccatalyst', 'runtime-linuxbionic') }}
 - name: isNotSpecificPlatformOnlyBuild
-  value: ${{ notin(variables['Build.DefinitionName'], 'runtime-wasm', 'runtime-wasm-libtests', 'runtime-wasm-non-libtests', 'runtime-ios', 'runtime-tvos', 'runtime-ioslike', 'runtime-iossimulator', 'runtime-tvossimulator', 'runtime-ioslikesimulator', 'runtime-android', 'runtime-androidemulator', 'runtime-catalyst', 'runtime-maccatalyst', 'runtime-linuxbionic', 'runtime-bionic') }}
+  value: ${{ notin(variables['Build.DefinitionName'], 'runtime-wasm', 'runtime-wasm-libtests', 'runtime-wasm-non-libtests', 'runtime-ioslike', 'runtime-ioslikesimulator', 'runtime-android', 'runtime-androidemulator', 'runtime-maccatalyst', 'runtime-linuxbionic') }}
 
 # We only run evaluate paths on runtime, runtime-staging and runtime-community pipelines on PRs
 # keep in sync with /eng/pipelines/common/xplat-setup.yml
```

Comment:
looks like we didn't keep this in sync :)

Reply:
though now that I think about it, not sure it makes sense to have them depend on evaluatepaths since they should always trigger I guess

Reply:
I think so. If we want to restrict it we can always add later.

Reply:
I only looked at a few commits in the past when only adding runtime-extra-platforms to the `isRunSmokeTestsOnly` section, it looks like the two diverged in https://github.com/dotnet/runtime/pull/64830